apps/ledger-live-mobile/src/components/RootNavigator/types/ReceiveFundsNavigator.ts

```diff
@@ -21,7 +21,7 @@ export type ReceiveFundsStackParamList = {
   [ScreenName.ReceiveAddAccountSelectDevice]: {
     accountId?: string;
     parentId?: string;
-    currency: CryptoCurrency | TokenCurrency;
+    currency: CryptoCurrency;
```

Comment:
Why not `CryptoOrTokenCurrency` here?

Reply:
see https://github.com/LedgerHQ/ledger-live/pull/2009